location/build.gradle.kts

```diff
@@ -23,13 +23,14 @@ kotlin {
     sourceSets {
         commonMain {
             dependencies {
-                implementation(project(":permissions", ""))
-                implementation(project(":logging", ""))
+                api(project(":permissions", ""))
+                api(project(":logging", ""))
+                api(project(":base", ""))
```

Comment:
why switch to `api`?

Reply:
Because for some reason without that when I build ios code (from Xbike) I get error
```
No matching variant of com.splendo.location:0.2.0 was found. The consumer was configured to find a usage of 'kotlin-api' of a component...
```

Reply:
This exports all methods of these libraries as part of this library.

Reply:
Do you know how can I fix that error without using api?

Reply:
Could it be by doing:
```
kotlin {
    android {
        publishAllLibraryVariants()
        publishLibraryVariantsGroupedByFlavor = true // This line
    }
}
```
(I am out of my waters here)

Reply:
resolved now, according to standup meeting.